cfg/CFG.h

```diff
@@ -29,7 +29,7 @@ class BlockExit final {
     BasicBlock *elseb;
     core::LocOffsets loc;
     bool isCondSet() {
-        return cond.variable._name.id() >= 0;
+        return cond.variable.exists();
```

Comment:
This is one consequence of this change. It used to be that cfg/builder/ was making a distinction between "unset" vs "set to noVariable". The only place where it was using "set to noVariable" was for unconditional branches, where it was just a convention. I've made that explicit by introducing a distinguished `<unconditional>` variable for those cases.

This has the benefit that the code reads more naturally.